chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -370,9 +374,26 @@ impl PeerManagerActor {
         client_addr: Recipient<NetworkClientMessages>,
         view_client_addr: Recipient<NetworkViewClientMessages>,
         genesis_id: GenesisId,
+    ) -> anyhow::Result<Self> {
+        PeerManagerActor::new_with_clock(
+            store,
+            config,
+            client_addr,
+            view_client_addr,
+            genesis_id,
+            time::Clock::real(),
```

Comment:
+1, in fact we should make new() accept the clock directly (so, rename new_with_clock -> new), please add a TODO.